scripts/run-sphinx.sh

```diff
@@ -5,9 +5,14 @@ SCRIPT_PATH=$(dirname $BASH_SOURCE)
 AMICI_PATH=$(cd $SCRIPT_PATH/.. && pwd)
 
 cd ${AMICI_PATH}/documentation
-source ../build/venv/bin/activate
-pip3 install -r ${AMICI_PATH}/documentation/rtd_requirements.txt
+python3 -m venv ${AMICI_PATH}/doc-venv --clear
+source ${AMICI_PATH}/doc-venv/bin/activate
+python -m pip install --upgrade --no-cache-dir pip
+python -m pip install git+https://github.com/readthedocs/readthedocs-sphinx-ext
+python -m pip install --upgrade --no-cache-dir Pygments==2.3.1 setuptools==41.0.1 docutils==0.14 mock==1.0.1 pillow==5.4.1 alabaster==0.7.12 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<1.1
```

Comment:
Move to requirements file? if leaving here, the parts with `<` / `>` will need quotes.

Reply:
thats all the stuff that rtd installs. I believe we can remove all of that